recipes/apu-label-propagation/meta.yaml

```diff
@@ -0,0 +1,28 @@
+{% set version = "1.2" %}
+
+package:
+  name: apu-label-propagation
+  version: {{ version }}
+
+source:
+  url: https://github.com/anuprulez/apu_label_propagation/archive/refs/tags/{{ version }}.tar.gz
+  sha256: bc2a108a706ce16f628b3e805902fa29c6a51c335891cc20a394f8cfec87ea88
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('apu-label-propagation') }}
```

Comment:
How stable is the ABI/API?

You might want to add `{{ pin_subpackage("myrecipe", max_pin="x.x.x") }}` if its really unstable.